samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/SubmitToGradingActionListener.java

```diff
@@ -707,28 +707,40 @@ private void prepareItemGradingPerItem(ActionEvent ae, DeliveryBean delivery,
 		case 15: // CALCULATED_QUESTION
 		case 16: //IMAGEMAP_QUESTION 	
 		case 11: // FIN
-			boolean addedToAdds = false;
 			for (int m = 0; m < grading.size(); m++) {
 				ItemGradingData itemgrading = grading.get(m);
 				itemgrading.setAgentId(AgentFacade.getAgentString());
 				itemgrading.setSubmittedDate(new Date());
 			}
+			int fake=-1;
```

Comment:
can you choose a better variable name here? what is the purpose of the variable?

Reply:
changed to fakitemgrading